src/app/globals.css

```
@@ -1,15 +1,53 @@
+
+  --color-ot-gray-100: #f7f7f7;
+  --color-ot-gray-200: #f0f0f0;
+  --color-ot-gray-300: #e8e8e8;
+  --color-ot-gray-400: #e1e1e1;
+  --color-ot-gray-500: #d9d9d9;
+  --color-ot-gray-600: #aeaeae;
+  --color-ot-gray-700: #828282;
+  --color-ot-gray-800: #575757;
+  --color-ot-gray-900: #2b2b2b;
+
+  --color-ot-text: #fafaf8;
```

Stylelint (17.2.0)

[error] Unexpected empty line before custom property (custom-property-empty-line-before)

(custom-property-empty-line-before)